modules/integration/tests-integration/tests-backend/pom.xml

```diff
@@ -527,6 +527,11 @@
             <groupId>org.wso2.carbon.identity.framework</groupId>
             <artifactId>org.wso2.carbon.user.mgt.stub</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.user.mgt</artifactId>
+            <version>${carbon.identity.framework.version}</version>
```

Comment:
please add this dependency to parent pom with the version if not already there.

Reply:
The version is taken from the parent pom.

Reply:
@menakaj if the version is there in the parent pom (in dependency management), it doesn't need to be included here. Refer the other dependencies.

Reply:
I checked this. If we do not add the dependency version, it fails with a ClassNotFound error.